test/integration/039_config_tests/test_configs.py

```diff
@@ -316,3 +318,33 @@ def test_postgres_config_with_get_default(self):
         self.assertEqual(len(results), 1)
         self.assertEqual(str(results[0].status), 'error')
         self.assertIn('column "default_value" does not exist', results[0].message)
+
+
+class TestConfigUtils(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "config_039"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def project_config(self):
```

Comment:
Should the test be a unit test that only test `get_project_config`'s function? This would work but I feel like it is harder to know exactly what is being tested if just reading the code.